compose/foundation/foundation/src/androidMain/kotlin/androidx/compose/foundation/text/TextFieldPointerModifier.android.kt

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2023 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package androidx.compose.foundation.text
+
+import androidx.compose.foundation.interaction.MutableInteractionSource
+import androidx.compose.foundation.text.selection.TextFieldSelectionManager
```

Comment:
Not sure that I tested it correctly, but I see that we select entire words instead of symbols:

https://github.com/JetBrains/compose-multiplatform-core/assets/5963351/41c0854d-7c07-4ee3-9b6d-42671cfeeb7d

I tested it on desktop, because I still has difficulties to run iOS from core repo - I could made a mistake somewhere (there some misconfiguration for Intel CPU, will look later)

Reply:
If I understood correctly, this logic remained unchanged
On iOS, there is no way to select text this way (without a mouse), usually selection handles dragging are in use.
So, I guess we should research this behavior and see if this is wrong or not in the separate issue